package.json

```diff
@@ -47,10 +48,12 @@
     "changelog": "standard-version",
     "inspect": "node --inspect dist/index.js",
     "release": "yarn reset && yarn test && yarn changelog",
+    "clean": "trash dist",
     "reset": "git clean -dfx && git reset --hard && yarn",
     "ci": "yarn test",
     "validate:dependencies": "yarn audit --groups dependencies",
-    "license-validate": "node-license-validator -p -d --allow-licenses MIT MIT/X11 BSD BSD-3-Clause BSD-2-Clause ISC Apache Apache-2.0 Unlicense --allow-packages cycle",
+    "validate": "license-checker --production --onlyAllow MIT;MIT/X11;BSD;BSD-3-Clause;BSD-2-Clause;ISC;Apache;Apache-2.0;Unlicense",
+    "validate-summary": "license-checker --summary --production --onlyAllow MIT;MIT/X11;BSD;BSD-3-Clause;BSD-2-Clause;ISC;Apache;Apache-2.0;Unlicense",
```

Comment:
Why does validate only validate licenses? And what is validate-summary?

Reply:
btw: `$ license-checker --production --onlyAllow MIT;MIT/X11;BSD;BSD-3-Clause;BSD-2-Clause;ISC;Apache;Apache-2.0;Unlicense 
Package "cliui@7.0.4" is licensed under "ISC" which is not permitted by the --onlyAllow flag. Exiting.
`

Reply:
Still open question is why we have license-checker involved - what is the enhancement against:
`"validate": "yarn audit --level moderate --groups dependencies"`